ejcIMC.py

```diff
@@ -34,6 +53,7 @@ def whileErrorNumbers(validation):
         value2 = numbersValidation(weigth)
         IMC = imcCalculate(value, value2)
         print('señor/señora', name, ' su indice de masa muscular es de: ', IMC )
+        yourImc(IMC)
         if validation == value:
```

Comment:
please change the position of this conditional, also make it in a function cuz u need validate value and value2

Reply:
ok i did solve